### PR TITLE
Feature#115/create homepage

### PR DESCRIPTION
**Pull request by rakhshanda-rashid**

### CODE REVIEW

File: src/client/components/RegisterButton/RegisterButton.css [hunk 1]
```diff
@@ -2,12 +2,11 @@
     border-radius: 40px;
     background-color:white;
     border: 1px solid black;
-    width: 215px;
-    height: 80px;
+    width: 150px;
```
Comment (laila1222) [inline src/client/components/RegisterButton/RegisterButton.css]: I suggest using rem instead of pixels for more flexibility and responsiveness.

File: src/client/components/RegisterButton/RegisterButton.css [hunk 2]
```diff
@@ -2,12 +2,11 @@
...
+    height: 45px;
```
Comment (laila1222) [inline src/client/components/RegisterButton/RegisterButton.css]: I suggest using rem instead of pixels for more flexibility and responsiveness.

File: src/client/components/RegisterButton/RegisterButton.css [hunk 3]
```diff
@@ -2,12 +2,11 @@
...
     color:#2D2E36;
-    font-family: "Open Sans";
-    font-size: 25px;
-    font-weight: bold;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;;
+    font-size: 19px;
```
Comment (laila1222) [inline src/client/components/RegisterButton/RegisterButton.css]: I suggest using rem instead of pixels for more flexibility and responsiveness.

File: src/server/seeds/006_00_annotations.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+          description: 'something new',
+          area: `{
+            "width": 20,
+            "height": 20
+          }`,
+          fk_screenshot_id: 1
+        },
+      ]);
+    });
+};
```
Comment (laila1222) [inline src/server/seeds/006_00_annotations.js]: This file is not related to create homepage, I suggest removing it from this PR.

**Review by laila1222 — APPROVED** [review]
Overall looks good! 👍 
There are a few things that needs to be fixed with the small components, like footer, adjusting the register button, and moving the toggle button signs, fixing the context-text and responsive version, but the page is put together very well! 

There are also some files from the backend on this pull request, that shouldn't be here in this pull request. (or else you'll end of with loads of merge conflicts)

File: src/server/api/controllers/users.controller.js [hunk 5]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (elilovna) [inline src/server/api/controllers/users.controller.js]: You don't need empty lines here

File: src/client/components/HomePage/HomePage.css [hunk 6]
```diff
@@ -0,0 +1,55 @@
+
+html, body{
+    height: 100%;
+    margin:0;
+    padding: 0;
+} */
```
Comment (elilovna) [inline src/client/components/HomePage/HomePage.css]: you probably want to delete `*/`

File: src/client/components/HomePage/HomePage.component.js [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+         hyfLogo={hyfLogo}
+         codeartLogo={codeartLogo}
+         theme={themeContent}
+         />
+      </div> 
+      <div className='footer-wrapper'>
+      <Footer />
+      </div>
+  </div>
+
```
Comment (elilovna) [inline src/client/components/HomePage/HomePage.component.js]: You don't need empty lines here

File: src/server/api/controllers/annotations.controller.js [hunk 8]
```diff
@@ -1,11 +1,15 @@
+
+
```
Comment (elilovna) [inline src/server/api/controllers/annotations.controller.js]: empty lines

File: src/client/components/HomePage/HomePage.css [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+    margin-left:4rem;
+    margin-right: 4rem;
+    margin-top:-1rem;
+  } 
+  .image-wrapper {
+    width: 95%;
+    margin-left:2rem;
+    margin-right: 2rem;
+    /* margin: 0 auto;
+    padding-top: 1.5rem; */
```
Comment (elilovna) [inline src/client/components/HomePage/HomePage.css]: it is better to delete commented code

File: src/server/api/controllers/annotations.controller.js (hunk 8 above)
Comment (rakhshanda-rashid) [reply src/server/api/controllers/annotations.controller.js]: hi, can you mention the lines which are empty

Comment (elilovna) [reply src/server/api/controllers/annotations.controller.js]: hi,  1 and 2

**Comment by laila1222** [comment]
New branch is created that is integrating these changes.
